### PR TITLE
Set default background for playlist web ui

### PR DESCRIPTION
**Pull request by sangwoo108**
Resolves https://github.com/brave/brave-browser/issues/31844

image

## Submitter Checklist:

- [ ] I confirm that no [security/privacy review is needed](https://github.com/brave/brave-browser/wiki/Security-reviews) and no other type of reviews are needed, or that I have [requested](https://github.com/brave/reviews/issues/new/choose) them
- [ ] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally:
  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

## Test Plan:
* Set to use dark mode
* Open playlist and see the background color

### CODE REVIEW

File: components/playlist/browser/resources/playlist.html [hunk 1]
```diff
@@ -14,6 +14,7 @@
 
   body {
     margin: 0;
+    background-color: var(--leo-color-container-background);
```
Comment (fallaciousreasoning) [inline components/playlist/browser/resources/playlist.html:17]: This is 100% fine as is, but thought I'd let you know - there's a [createGlobalStyle](https://styled-components.com/docs/api#createglobalstyle) utility in styled components

File: components/playlist/browser/resources/videoFrameContents.tsx [hunk 2]
```diff
@@ -8,10 +8,6 @@ import * as React from 'react'
 import { render } from 'react-dom'
 import { Provider } from 'react-redux'
 
-import BraveCoreThemeProvider from '../../../common/BraveCoreThemeProvider'
-import Theme from 'brave-ui/theme/brave-default'
```
Comment (fallaciousreasoning) [inline components/playlist/browser/resources/videoFrameContents.tsx:12]: :+1:

**Review by fallaciousreasoning — APPROVED** [review]
LGTM!

File: components/playlist/browser/resources/playlist.html (hunk 1 above)
Comment (sangwoo108) [reply components/playlist/browser/resources/playlist.html:17]: Thanks @fallaciousreasoning ! I tried that but somehow it didn't work but not sure where I made a mistake. Could you take a look at https://github.com/brave/brave-core/compare/sko/pl-background-test?expand=1 ?

image

Comment (fallaciousreasoning) [reply components/playlist/browser/resources/playlist.html:17]: Oh weird, I would've thought that would work :laughing: - I'll have a play and see if I can work it out, but this is fine as is :smile:

Comment (fallaciousreasoning) [reply components/playlist/browser/resources/playlist.html:17]: (I'm marking this as resolved, while I faff around with it)

Comment (sangwoo108) [reply components/playlist/browser/resources/playlist.html:17]: could this be related to CSP of chrome-untrusted:// frame?

**Pull request merged**